.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,122 @@
+name: Bug Report
+description: Report a problem and help us fix it.
+labels: ["kind/bug"]
+body:
+  - type: dropdown
+    id: Component
+    attributes:
+      label: Component
+      description: In which component did the issue occur?
+      options:
+        - Not sure-
+        - Operate-
+        - Optimize-
+        - Tasklist-
+        - Identity-
+        - Zeebe-
+        - C8Run-
+      default: 0
+    validations:
+      required: true
+  - type: dropdown
+    id: severity
+    attributes:
+      label: Severity
+      description: What is the severity of the bug?
+      options:
+        - Low-
+        - Medium-
+        - High-
+        - Critical-
+      default: 0
+    validations:
+      required: true
+  - type: dropdown
+    id: likelihood
+    attributes:
+      label: Likelihood
+      description: How likely is it to occur?
```

Comment:
Same here as above. If it's filled out by customers/users, they will most probably set likelihood higher, as they are experiencing this issue more often than the global customer base 🤔